molecule/drupal8-db-dump/converge.yml

```diff
@@ -0,0 +1,141 @@
+---
+- name: Converge
+  hosts: instance
+  become: true
+  pre_tasks:
+    - name: Add ppa:ondrej/php repository
+      apt_repository:
+        repo: ppa:ondrej/php
+      when: ansible_os_family == 'Debian'
+    - name: Import remi GPG key.
+      rpm_key:
+        key: "https://rpms.remirepo.net/RPM-GPG-KEY-remi2018"
+        state: present
+      when: ansible_os_family == 'RedHat'
+    - name: Add remi-php74 repository
+      yum:
+        name: https://rpms.remirepo.net/enterprise/remi-release-{{ ansible_distribution_major_version }}.rpm
+        state: present
+      when: ansible_os_family == 'RedHat'
+    - name: Enable module php7.4 for Centos8
+      dnf:
+        name: '@php:remi-7.4'
+        state: present
+      when: ansible_os_family == 'RedHat' and ansible_distribution_major_version == '8'
+    - name: Install python3 Psycopg2
+      apt:
+        name: python3-psycopg2
+      when: ansible_os_family == 'Debian'
+    - name: "Set ansible_python_interpreter for Ubuntu 20"
+      set_fact:
+        ansible_python_interpreter: "/usr/bin/python3"
+      when: ansible_distribution == 'Ubuntu' and ansible_distribution_major_version | int >= 18
+  tasks:
+    - name: Include geerlingguy.php
+      include_role:
+        name: geerlingguy.php
+      vars:
+        php_enablerepo: "remi-php74"
+        php_default_version_debian: "7.4"
+        php_install_recommends: false
+        php_enable_php_fpm: true
+        php_webserver_daemon: "nginx"
+        php_date_timezone: "Europe/Paris"
+        php_expose_php: false
+        php_packages_extra: "{{ (ansible_os_family == 'Debian') | ternary(['php7.4-pgsql'], ['php-pgsql']) }}"
```

Comment:
Drush commands complains about missing apcu.so, might be missing here. Valid for other scenarios (until factorization).

Reply:
Apcu n'est pas obligatoire